src/ar/mangalek/src/eu/kanade/tachiyomi/extension/ar/mangalek/Mangalek.kt

```diff
@@ -35,38 +45,30 @@ class Mangalek :
     override val useLoadMoreRequest = LoadMoreStrategy.Always
     override val chapterUrlSuffix = ""
 
-    private val defaultBaseUrl = mangalekUrl
-    override val baseUrl by lazy { getPrefBaseUrl() }
+    override val baseUrl by lazy { getMirrorPref() }
```

Comment:
make it something like [this](https://github.com/keiyoushi/extensions-source/blob/59c3a8d9acde0e967d31320ea93860bfa49bb542/src/vi/ungtycomics/src/eu/kanade/tachiyomi/extension/vi/ungtycomics/UngTyComics.kt#L31)

This is to make sure the automod on github is able detect all the mirrors for the site

Reply:
Ook

Reply:
"CI" is not a country code, and should not be set to "AR". CI stands for Continuous Integration, which is used to detect the mirrors.